backup/backupd.c

```diff
@@ -894,6 +896,8 @@ static int cmd_apply_message(struct dlist *dl)
     struct sync_msgid_list *guids = sync_msgid_list_create(0);
     struct dlist *di;
```

Comment:
I am against such changes.  The closest the variable declaration are to the first variable use, the easiest the code is to read and understand. Writing on one line both declaration and definition is just less text in total.  Yes, other places in the code do pit the declarations to the very top, but I do not like such places, too.